mempool/src/shared_mempool/peer_manager.rs

```diff
@@ -72,14 +72,14 @@ impl Ord for BatchId {
     }
 }
 
-/// Txn broadcast-related info for a given remote peer
+/// Txn broadcast-related info for a given remote peer.
 #[derive(Clone)]
 pub struct BroadcastInfo {
-    // sent broadcasts that have not yet received an ACK
+    // Sent broadcasts that have not yet received an ack.
     pub sent_batches: BTreeMap<BatchId, SystemTime>,
-    // broadcasts that have received a retry ACK and are pending a resend
+    // Broadcasts that have received a retry ack and are pending a resend.
```

Comment:
Here's another debatable `ACK` vs `ack`, but really I don't think it matters either way as long as you're making them all consistent.